addon/components/hyper-table.js

```diff
@@ -45,13 +47,16 @@ export default Component.extend({
   _orderedFilteredColumns: computed(
     '_columns',
     '_availableColumnsKeyword',
+    '_activeColumnCategory',
     function() {
       let columns = A(this._columns);
 
-      if (!isEmpty(this._availableColumnsKeyword)) {
-        let reg = RegExp(this._availableColumnsKeyword, 'i');
-        columns = A(columns.filter((x) => reg.test(x.title)));
-      }
+      columns = A(columns.filter((x) => {
+        const hasKeyword = x.title.toLowerCase().indexOf(this._availableColumnsKeyword.toLowerCase()) >= 0;
```

Comment:
`_availableColumnsKeyword` can be empty like the activeColumn, no ? Maybe do the same